cmd/collector/app/root_span_handler_test.go

```diff
@@ -43,27 +44,27 @@ func TestHandleRootSpan(t *testing.T) {
 
 	// Testing non-root span
 	span := &model.Span{References: []model.SpanRef{{SpanID: model.NewSpanID(1), RefType: model.ChildOf}}}
-	processor(span)
+	processor(span, context.TODO())
```

Comment:
I think you want context.Background() in the tests - TODO() implies you want to change it in the future